test/electron/fixtures/app/renderer.js

```diff
@@ -7,11 +7,17 @@ const Bugsnag = require('@bugsnag/electron')
 Bugsnag.start(window.RunnerAPI.rendererConfig)
 const startupTimestamp = Date.now()
 
-function emulateOnlineStatus (online) {
+function emulateOnlineStatus(online) {
   Object.defineProperty(window.navigator, 'onLine', { value: online, configurable: true })
   window.dispatchEvent(new window.Event(online ? 'online' : 'offline'))
 }
 
+document.getElementById('renderer-process-crash').onclick = () => {
+  setTimeout(() => {
+    window.RunnerAPI.renderProcessCrash()
+  }, 10)
```

Comment:
It just needs to be async. Without the timeout, the click event would throw an exception in the automation framework since the listener would not be able to tell that the click "succeeded".